Parsely/wp-parsely
# Fix recrawl string i18n

## Description

Pull the wrapping `p` element out of the translatable string in the admin page script.

Fixes #283

## Motivation and Context

Adhere to coding standards.

## How Has This Been Tested?

* Apply this branch
* Load `/wp-admin/options-general.php?page=parsely`
* The recrawl notices should behave as before

## Screenshots (if appropriate):

## Types of changes
Bug fix (non-breaking change which fixes an issue)
